backend/channels.py

```diff
@@ -621,14 +736,40 @@ async def publish_channel_muxes(config):
                     else:
                         logger.info("    - Updating existing MUX '%s' for channel '%s' with stream '%s'", mux_uuid,
                                     result.name, source.playlist_stream_url)
+                    
```

Comment:
I feel like this whole section is not meant to be here.
I don't think iptv_epgid does anything for us in TVH. Unless I have completely forgotten something, this did not end up being needed. The mapping of the services to the EPG is done with the channel number.